src/main/resources/templates/faq.html

```diff
@@ -118,6 +118,10 @@ <h2 class="page-title heading-icon clearfix"><img th:src="@{/images/map-arrow-or
 
                     <p>a. Please complete the word fragments based only on information given to you in the brief
                         stories.</p>
+
+                    <h2>Reference:</h2>
+                    <p>Hallion, L. S., & Ruscio, A. M. (2011). A meta-analysis of the effect of cognitive bias modification
```

Comment:
You can't put an "&" directly into the html.  It is a reserved character.  You need to replace this with "&amp;"